src/Form/ParagraphsPasteForm.php

```diff
@@ -156,17 +162,24 @@ public static function pasteSubmit(array $form, FormStateInterface $form_state)
    *
    * @param array $data
    *   Pasted data.
+   * @param array $plugin_configuration
+   *   Plugin configuration.
    *
    * @return array
    *   Enriched data.
    */
-  public static function traverseData(array $data) {
+  public static function traverseData(array $data, array $plugin_configuration) {
+    /** @var \Drupal\paragraphs_paste\ParagraphsPastePluginManager $plugin_manager */
     $plugin_manager = \Drupal::service('plugin.manager.paragraphs_paste.plugin');
     $results = [];
 
     // Enrich pasted data with plugins.
     foreach ($data as $value) {
-      $results[] = (object) ['plugin' => $plugin_manager->getPluginFromInput($value), 'value' => $value];
+      $plugin = $plugin_manager->getPluginFromInput($value);
+      if ($plugin && ($property_path = $plugin_configuration[$plugin->getPluginId()])) {
```

Comment:
Please move check for property path into getPluginFromInput() or isApplicable(), it could well be that there are two plugins applicable for text data, but only one has propertypath configured